### PR TITLE
Fix group page related issues

### PR DESCRIPTION
**Pull request by herflis**

### CODE REVIEW

File: examples/sn-dms-demo/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -295,15 +297,6 @@ class DashboardComponent extends React.Component<
...
-                        <ContentTypes />
-                      </Route>
-                      <Route path="/contenttemplates">
-                        <ContentTemplates />
-                      </Route>
-                      <Route path="/settings">
-                        <Settings />
-                      </Route>
 
                       {/* <Redirect to="/documents" /> */}
```
Comment (B3zo0) [inline examples/sn-dms-demo/src/pages/Dashboard.tsx]:
```suggestion
```

File: packages/sn-controls-react/package.json [hunk 2]
```diff
@@ -44,7 +44,6 @@
     "@sensenet/pickers-react": "^1.0.2",
     "lodash.debounce": "^4.0.8",
     "material-ui-pickers": "^2.2.1",
-    "moment": "^2.24.0",
```
Comment (B3zo0) [inline packages/sn-controls-react/package.json]: date-io depends on moment. I'd let that here or add that as a peer dependency.

File: packages/sn-pickers-react/src/ListPicker/ListPicker.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ export function ListPickerComponent<T extends GenericContent = GenericContent>(p
 
   const onItemDoubleClickHandler = (event: React.MouseEvent, node: T) => {
     event.preventDefault()
-    props.onNavigation && props.onNavigation()
+    props.onNavigation && props.onNavigation(node.Path)
```
Comment (B3zo0) [inline packages/sn-pickers-react/src/ListPicker/ListPicker.tsx]: this should be handled below but it is already fixed in another pr. #180

File: examples/sn-dms-demo/src/__tests__/Sensenet.test.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
-// import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (herflis) [reply examples/sn-dms-demo/src/__tests__/Sensenet.test.tsx]: all the dms component test should be reviewed in the near future. I'll leave this till the review

File: examples/sn-dms-demo/src/components/ActionMenu/__tests__/ActionMenu.test.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // import { Store } from '@sensenet/redux'
```
[the remark this answers is not in the document]
Comment (herflis) [reply examples/sn-dms-demo/src/components/ActionMenu/__tests__/ActionMenu.test.tsx]: Leave it later, all these test files should be reviewed/refactored/removed from the dms

File: packages/sn-controls-react/package.json (hunk 2 above)
Comment (herflis) [reply packages/sn-controls-react/package.json]: ok

**Pull request merged**